keyboards/elcantorhs/readme.md

```diff
@@ -0,0 +1,30 @@
+# El Cantor HS
+
+![El Cantor HS](https://thumb.cloud.mail.ru/weblink/thumb/xw1/VtFZ/jwSN7r6BB)
+
+The El Cantor HS keyboard is a 43 key diodeless split keyboard with support for hotswap sockets, designed with simplicity in mind. It is based on the popular [cantor](https://github.com/diepala/cantor), which is inspired on the popular [corne](https://github.com/foostan/crkbd), [ferris](https://github.com/pierrechevalier83/ferris) and [sweep](https://github.com/davidphilipbarr/Sweep) keyboards, aiming to provide a more ergonomic (stronger column stagger) corne-like layout with a simple, easy to assemble and cheap design.
```

Comment:
No need to elaborate on Cantor's design influences, this readme.md is all about the El Cantor HS 😉
```suggestion
The El Cantor HS keyboard is a 44 key diodeless split keyboard with support for hotswap sockets, designed with simplicity in mind. Based on the popular [Cantor](https://github.com/diepala/cantor) keyboard, this keyboard features a more aggressive column stagger than the afore mentioned, whilst aiming to simplify assembly and improve cost efficiency.
```